src/libcore/lib.rs

```diff
@@ -23,12 +23,6 @@
 //! nor does it provide concurrency or I/O. These things require
 //! platform integration, and this library is platform-agnostic.
 //!
-//! *It is not recommended to use the core library*. The stable
```

Comment:
We might still want to leave something here saying that most users will want to use libstd instead?